matea-pjanic/assignments/liferay-app/sdk/portlets/music-portlet/docroot/js/main.js

```diff
@@ -0,0 +1,57 @@
+AUI().add(
+	'search-music',
+	function(A) {
+		var SearchMusic = A.Component.create(
+			{
+				AUGMENTS: [Liferay.PortletBase],
+
+				EXTENDS: A.Base,
+
+				NAME: 'search-music',
+
+				prototype: {
+					initializer: function(config) {
+						var instance = this;
+
+						var button = instance.byId('search');
+
+						if (button) {
+							button.on('click', instance.ajaxCall);
+						}
+
+						var input = instance.byId('music');
+
+						if (input) {
+							input.on('key', instance.ajaxCall, 'enter');
+						}
+					},
+
+					ajaxCall: function() {
```

Comment:
1. Private function should start with underline
1. ajaxCall is too generic, should be ```_getMusic()```
1. Event callback function should have argument ```'event'```.
1. Each module property function should start with ```var instance = this```